### ISSUE TITLE
Release version for Ubuntu 18.04 crashes

### ISSUE THREAD

**Issue by frederik-elwert** [DARIAH-DE/TopicsExplorer#78]
When trying to launch the released binary for Ubuntu 18.04 (v0.1.6), the application crashes and I get this output:

```
$ ./DARIAH\ Topics\ Explorer 
[1113/104631.206092:WARNING:stack_trace_posix.cc(648)] Failed to open file: /tmp/.gl6TqhhV (deleted)
  Error: Datei oder Verzeichnis nicht gefunden
Received signal 11 SEGV_MAPERR 000000000000
#0 0x7fb5fba219a5 <unknown>
#1 0x7fb5fa761501 <unknown>
#2 0x7fb5fba21d3d <unknown>
#3 0x7fb6279c7f20 <unknown>
  r8: 0000000000000001  r9: 000000000355cca0 r10: 0000000000000002 r11: 00007fb606c3f000
 r12: 000000000355cd00 r13: 00000000021d9b50 r14: 0000000000000000 r15: 00007fb606d30280
  di: 0000000000001f01  si: 00007fb606d32210  bp: 00007fb5682d1e60  bx: 0000000000000001
  dx: 0000000003585600  ax: 0000000003585600  cx: 0000000000000000  sp: 00007ffc3ae128d8
  ip: 0000000000000000 efl: 0000000000010206 cgf: 002b000000000033 erf: 0000000000000014
 trp: 000000000000000e msk: 0000000000000000 cr2: 0000000000000000
[end of stack trace]
Calling _exit(1). Core file will not be generated.
```

**Comment by frederik-elwert** [comment]
Interestingly, I get the same error when trying to install/launch the app using the source release and pipenv.

**Comment by frederik-elwert** [comment]
Almost the same error with current master. But now I also get an NSS error:

```
$ python topicsexplorer.py 
[1113/105853.641752:WARNING:stack_trace_posix.cc(648)] Failed to open file: /tmp/.glNtqIpp (deleted)
  Error: Datei oder Verzeichnis nicht gefunden
 * Serving Flask app "application.config" (lazy loading)
[30733:30768:1113/105853.754393:ERROR:nss_util.cc(750)] After loading Root Certs, loaded==false: NSS error code: -8018
Received signal 11 SEGV_MAPERR 000000000000
#0 0x7f8eb846c9a5 <unknown>
#1 0x7f8eb71ac501 <unknown>
#2 0x7f8eb846cd3d <unknown>
#3 0x7f8ee421ff20 <unknown>
  r8: 0000000000000001  r9: 00007f8ea8007f90 r10: 0000000000000002 r11: 00007f8ec3620000
 r12: 0000000003e14ac0 r13: 0000000003fb7770 r14: 0000000000000000 r15: 00007f8ec3711280
  di: 0000000000001f01  si: 00007f8ec3713210  bp: 000000000408d580  bx: 0000000000000001
  dx: 0000000003f6f140  ax: 0000000003f6f140  cx: 0000000000000000  sp: 00007ffd28648d68
  ip: 0000000000000000 efl: 0000000000010206 cgf: 002b000000000033 erf: 0000000000000014
 trp: 000000000000000e msk: 0000000000000000 cr2: 0000000000000000
[end of stack trace]
Calling _exit(1). Core file will not be generated.
```

**Comment by severinsimmler** [comment]
Thank you for this bug report. Both error messages are related to PyQt5 (or more precisely: Chromium), whose web engine is used to render the UI. The UI itself is built with web technologies (basically Flask and some JavaScript), so the PyQt5 component is more or less optional (but _actually_ wanted for the desktop application feel). We are currently working on v2.0 with some general improvements (release will be definitely in 2018), hopefully fixing this one too. Until then you could use your default browser to render the UI (= running the app).

**TL;DR**
As a workaround, you could run the application from source with:
```
$ python topicserxplorer.py --browser
```
which will launch the app in your default browser.

**Comment by severinsimmler** [comment]
v2.0 has been [released](https://github.com/DARIAH-DE/TopicsExplorer/releases/tag/v2.0). This error should no longer occur, if it does, please open a new issue. Thank you!
